containers/line-items/index.js

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from '@apollo/client'
```

Comment:
I don't think we are using this whole component at the moment anywhere. Should we delete it?